pkgs/development/compilers/llvm/13/default.nix

```diff
@@ -247,11 +247,18 @@ let
                else stdenv;
     };
 
-    libcxxabi = callPackage ./libcxxabi {
-      inherit llvm_meta;
-      stdenv = if stdenv.hostPlatform.useLLVM or false
+    libcxxabi = let
+      stdenv_ = if stdenv.hostPlatform.useLLVM or false
                then overrideCC stdenv buildLlvmTools.clangNoLibcxx
                else stdenv;
+      cxx-headers = callPackage ./libcxx {
+        inherit llvm_meta;
+        stdenv = stdenv_;
+        isCxxHeaders = true;
```

Comment:
I've seen `headersOnly` as the argument for triggering a headers only build  a couple times, might be nice to align with that convention?

Reply:
I've now changed it to `headersOnly`.